frontend/src/metabase/routes.jsx

```diff
@@ -384,7 +387,7 @@ export const getRoutes = store => (
         })
       }
     />
-    <Redirect from="/dash/:dashboardId" to="/dashboard/:slug" />
+    <Redirect from="/dash/:dashboardId" to="/dashboard/:dashboardId" />
```

Comment:
This wasn't working.

![image](https://user-images.githubusercontent.com/1937582/165925015-8ed357c2-1623-4cb1-bf70-7b7e0617dacb.png)